### ISSUE TITLE
docker image 2.7.0 not working

### ISSUE THREAD

**Issue by drummerwolli** [vale-cli/vale#305]
Hi, just trying out vale. looks promising so far. only, i had start difficulties with the newest docker image:

1. clone the [boilerplate project](https://github.com/errata-ai/vale-boilerplate)
2. go into the cloned directory
3. run `docker run --rm -v $(pwd)/styles:/styles --rm -v $(pwd):/docs -w /docs jdkato/vale:v2.7.0 .`

error message: `lstat : no such file or directory`

with docker image `v2.3.3` it works. 

possibly related: https://github.com/errata-ai/vale/issues/303

**Comment by jdkato** [comment]
I'm unable to reproduce this:

```console
$ docker run --rm -v $(pwd):/docs -w /docs jdkato/vale:v2.7.0 .

 README.md
 13:20   warning  'extremely' is a weasel word!  write-good.Weasel    
 15:120  warning  'However' is too wordy.        write-good.TooWordy  
 27:6    error    'is' is repeated!              Vale.Repetition      
 27:6    warning  'is' is repeated!              write-good.Illusions 

✖ 1 error, 3 warnings and 0 suggestions in 2 files.
```

**Comment by drummerwolli** [comment]
mh, weird. here is my system information:

```
$ lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 20.04.1 LTS
Release:	20.04
Codename:	focal
```

```
$ docker --version                                                                           
Docker version 19.03.13, build 4484c46d9d
```

**Comment by jdkato** [comment]
The problem is the `.vale.ini`:

https://github.com/errata-ai/vale-boilerplate/blob/90751ea3949d498d7e53987d00d8c43504f52df0/.vale.ini#L2-L4

`Vocab = blog` should be `Vocab = Blog`.

This error message will be improved in the next release and `vale-boilerplate` has been fixed.

**Comment by th0ger** [comment]
@jdkato it would be great with some error handling, stating which file is not found.

**Comment by jdkato** [comment]
In the next version you'll see:

```console
E100 [vocab] Runtime error

Vocab 'blog' does not exist

Execution stopped with code 1.
```

**Comment by drummerwolli** [comment]
thx for looking into this @jdkato !
